gradle/scripts/lib/java-rpc-thrift.gradle

```
@@ -21,6 +21,9 @@ configure(projectsWithFlags('java')) {
                 group: 'Build',
                 description: "Compiles the ${sourceSet.name} .thrift files."
         ])
+        if (scope == 'main') {
+            project.sourcesJar.dependsOn(task)
+        }
```

```suggestion
        def sourceJarTask = projects.tasks.findByName(sourceSet.getTaskName('sources', 'jar'))
        if (sourceJarTask) {
            sourceJarTask.dependsOn(task)
        }
```

Please make a similar change to `java-rpc-proto.gradle`.